README.md

```diff
@@ -18,93 +18,15 @@ contributor should review our code of conduct at
 If you have any complaints please let us know by sending an email to
 info@womenrising.co and we will do our best to address them!
 
-### Getting Started
-In order to get started with this project please fork the repo and clone it to
-have it locally on your computer.
+## Getting Started
+Check out our [Contributing doc](https://github.com/womenrising/womenrising/blob/master/CONTRIBUTING.md) for more information on contributing to this project. In order to get started please fork the repo and clone it to have it locally on your computer.
 
-#### Development Environment
+### Development Environment
 
 If you have a mac and already have ruby set up, follow [these directions.](https://github.com/womenrising/womenrising#mac-osx)
```

Comment:
Ruby instructions 1-5 are basically "getting started with Ruby" and aren't really things that people who have done development with Ruby before should have to do. I wonder if they could be moved to a different file.  After all, this line says "already have ruby set up".  If you can't manage rvm and postgres on your own maybe you should do the docker thing.

Reply:
Good point. Though I can't follow both of these suggestions.